bitnami/apache/templates/ingress.yaml

```diff
@@ -31,9 +31,12 @@ spec:
             pathType: {{ .Values.ingress.pathType }}
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
+          {{- if .Values.ingress.extraPaths }}
+          {{- toYaml .Values.ingress.extraPaths | nindent 10 }}
```

Comment:
As a general comment, I would replace `toYaml` by `tplValues.render` function unless we are 100% sure that the values will never contain a template. The `tplValues.render` function performs a `tpl` apart from the `toYaml`. There are some cases where users could need to add a template in the values.yaml that with `toYaml` will not be rendered while with `tplValues.render` it will.